src/components/new-request/search.vue

```diff
@@ -1,22 +1,22 @@
 <template>
-  <v-container fluid id="new-request-container" class="copy-normal pa-10">
+  <v-container fluid id="search-container" class="copy-normal pa-10">
     <v-row no-gutters>
       <v-col cols="12" class="pt-0 font-weight-bold h6">
-        I need a name to: {{ request_action_cd }} / {{ location }} / {{ entity_type_cd }}
+        I need a name to: {{ getRequestActionCd }} / {{ getLocation }} / {{ entity_type_cd }}
       </v-col>
     </v-row>
 
-    <v-row class="mt-4" no-gutters>
-      <!-- request_action_cd -->
-      <v-col cols="12" md="4" lg="4">
+    <v-row class="mt-6">
+      <!-- Request Action -->
+      <v-col cols="12" md="6" class="py-0">
```

Comment:
`lg` was redundant.

This column is declared as "12" wide normally (small viewports are larger), and then when we hit a `md` size viewport, this column becomes "6" wide.